From b7a0dfa2beecfb0fd4f3d874a0a73dde0fa1b413 Mon Sep 17 00:00:00 2001
From: Han Xu <b45815@freescale.com>
Date: Fri, 5 Jun 2015 10:32:56 -0500
Subject: [PATCH] MLK-11057: ARM:dts: ECSPI Nor should be n25q032 on i.MX6UL
 ARM2

The ECSPI Nor on i.MX6UL ARM2 board is n25q032 rather than m25p80.

Signed-off-by: Han Xu <b45815@freescale.com>
---
 arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts b/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts
index d77531c..1ed444b 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts
@@ -88,10 +88,10 @@
 	pinctrl-0 = <&pinctrl_ecspi1_1 &pinctrl_ecspi1_cs_1>;
 	status = "okay";
 
-	flash: m25p80@0 {
+	flash: n25q032@0 {
 		#address-cells = <1>;
 		#size-cells = <1>;
-		compatible = "st,m25p32";
+		compatible = "st,n25q032";
 		spi-max-frequency = <20000000>;
 		reg = <0>;
 	};
-- 
1.8.0

